### DISCUSSION TITLE
Resource should open create page by default

### DISCUSSION THREAD

**Discussion by kimdigitary** [filamentphp/filament#12295] [Help] [answered]
### Package

Panel builder

### Package Version

v3.00

### How can we help you?

I have a Settings resource. I dont need a table for this resource. How do I open the create page directly? A create/edit page in one.
Somethiing like here below:
![image](https://github.com/filamentphp/filament/assets/142266990/4d5e01ed-a4f5-4d06-b2b9-16f6efb05c07)

**Comment by leandrocfe** [comment, marked as answer]
You could use this: https://filamentphp.com/plugins/filament-spatie-settings

**Reply by kimdigitary** [reply]
Thanks
